Lab_1.md

```diff
@@ -43,12 +43,142 @@ The last step
 
 #### Fizzbuzz
 * starting at one and incrementing by one to a number that is given, print out either the number, or if the number is divisible by 2, print "Fizz", and if the number is divisible by 3 print "Buzz" 
+~~~
+-----------------------
+Insert Desired Number   
+-----------------------       
+        |                     
+        V                     
+       / \                                                     |<-------------
+      /   \                                                    |             ^
+/  start a var at 1  \                                         |             |
+\              /                                               |           /  \
+      \   /                                                    |      \Add one to var/
+       \ / ----------------------------------------------------<             |       
+        |                                                                   No                        -------
+        |                                                       /                       \---------yes > End
+        |                                                       \Is Var euqal to desired Num/         -------
+  /Divide var by 2\                                             |
+ \Is there a remaineder?\----No---> /print fuzz\ ---------------^
```

Comment:
What about 6?  (I'm not asking for any changes, just wanting to bring up that there is often more to a problem than it initially appears.)